### PR TITLE
Check r.ParseForm() error in postContainersCreate().

### PR DESCRIPTION
**Pull request by NanXiao**
Signed-off-by: NanXiao <nan@chinadtrace.org>

### CODE REVIEW

File: api/handlers.go [hunk 1]
```diff
@@ -537,7 +537,10 @@ func getContainerJSON(c *context, w http.ResponseWriter, r *http.Request) {
 
 // POST /containers/create
 func postContainersCreate(c *context, w http.ResponseWriter, r *http.Request) {
-	r.ParseForm()
+	if err := r.ParseForm(); err != nil {
+		httpError(w, err.Error(), http.StatusInternalServerError)
```
Comment (xianlubird) [inline api/handlers.go]: StatusInternalServerError ？  Maybe 400 error is a nice choice ?

Comment (dongluochen) [inline api/handlers.go]: Agree with @xianlubird that `http.StatusBadRequest` should be returned on parse error.

Comment (NanXiao) [inline api/handlers.go]: @xianlubird & @dongluochen : Thanks very much for your comments! The commit has been updated.

Comment (allencloud) [inline api/handlers.go]: While all other err returned by r.ParseForm return StatusInternalServerError, except GET /events.

**Comment by dongluochen** [comment]
LGTM. 

Test failure on master is unrelated. I'll take a look.

**Comment by nishanttotla** [comment]
LGTM.

The network test failure was resolved in a separate PR.

**Pull request merged**